src/main/java/bio/terra/testrunner/common/HelmReleaseVersion.java

```diff
@@ -0,0 +1,42 @@
+package bio.terra.testrunner.common;
+
+import java.util.Optional;
+
+public class HelmReleaseVersion {
```

Comment:
nit: same as above. could you please add a description. maybe with a link to the helm repo where these things are getting pulled from?

Reply:
Added description to `HelmReleaseVersion`.